src/modules/AtakPluginModule.cpp

```diff
@@ -52,6 +52,10 @@ meshtastic_TAKPacket AtakPluginModule::cloneTAKPacketData(meshtastic_TAKPacket *
     } else if (t->which_payload_variant == meshtastic_TAKPacket_chat_tag) {
         clone.which_payload_variant = meshtastic_TAKPacket_chat_tag;
         clone.payload_variant.chat = {0};
+    } else if (t->which_payload_variant == meshtastic_TAKPacket_detail_tag) {
+        clone.which_payload_variant = meshtastic_TAKPacket_detail_tag;
+        clone.payload_variant.detail.size = t->payload_variant.detail.size;
+        memcpy(clone.payload_variant.detail.bytes, clone.payload_variant.detail.bytes, t->payload_variant.detail.size);
```

Comment:
memcpy is source, dest, size, yeah? Is the source and dest supposed to be the same here?

Reply:
Doh! Good eyes.